rtbkit/core/router/router_runner.cc

```diff
@@ -161,6 +164,9 @@ init()
                                       slowModeTimeout, amountSlowModeMoneyLimit);
     router->slowModeTolerance = slowModeTolerance;
     router->initBidderInterface(bidderConfig);
+    if(dableSlowMode) {
+       router->unsafeDisableMonitor();
```

Comment:
What seemed to be a good idea at first did not quite turn to be what we wanted I think.

The `unsafeDisableMonitor` also disables the **MonitorProviderClient**, which is the class responsible for sending various indicators to the monitor. For the router, the indicators are:

* Status of the SlaveBanker ;
* Connection to the PAL.

I don't think that we want to also disable these checks when disabling the slow mode, we only want to disable the monitorClient. Maybe the solution would be to add an other member function `unsafeDisableSlowmode` that will only disable the monitor client.